apps/web/src/components/search-result/index.tsx

```diff
@@ -146,14 +146,14 @@ function SearchResult(props: SearchResultProps) {
                 useEditorStore.getState().openSession(item.id, {
                   rawContent: item.rawContent,
                   force: true,
-                  activeSearchResultIndex: findSelectedMatchIndex(item, index)
+                  activeSearchResultId: match[0].id
```

Comment:
if there are multiple matches in the same search item, we only scroll to the first match, I don't think this should be a problem